src/net_processing.cpp

```diff
@@ -2482,6 +2482,15 @@ void ProcessMessage(
                       pfrom.nVersion.load(), pfrom.nStartingHeight,
                       pfrom.GetId(), (fLogIPs ? strprintf(", peeraddr=%s", pfrom.addr.ToString()) : ""),
                       pfrom.m_tx_relay == nullptr ? "block-relay" : "full-relay");
+            if (!pfrom.m_tx_relay) {
```

Comment:
I think after #19316 we should be able to check directly for the block-relay connection type?

Reply:
[Updated](https://github.com/bitcoin/bitcoin/pull/17428#issuecomment-678109126).